### PR TITLE
Model changes to house total active count of a few MOs

### PR DESCRIPTION
**Pull request by gautvenk**
Signed-off-by: Gautam Venkataramanan <gautam.chennai@gmail.com>

### CODE REVIEW

File: genie/MODEL/SPECIFIC/OBSERVER/total.mdl [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        member[localEP; type=scalar/UInt64]
+
+        # Total number of external EPs
+        member[extEP; type=scalar/UInt64]
+
+        # Total number of remote EPs
+        member[remoteEP; type=scalar/UInt64]
+
+        # Total number of EPGs
+        member[EPG; type=scalar/UInt64]
```
Comment (tomflynn) [inline genie/MODEL/SPECIFIC/OBSERVER/total.mdl:31]: convention is to always use lowercase for first character of each member. Can you update?

File: genie/MODEL/SPECIFIC/OBSERVER/total.mdl [hunk 2]
```diff
@@ -0,0 +1,63 @@
+module[observer]
+{
+    # Counter to hold total of MOs
+    class[TotalCounter;
```
Comment (tomflynn) [inline genie/MODEL/SPECIFIC/OBSERVER/total.mdl:4]: Can we rename to something like MoCounts or ModbCounts or something similar

File: genie/MODEL/SPECIFIC/OBSERVER/total.mdl (hunk 1 above)
Comment (gautvenk) [reply genie/MODEL/SPECIFIC/OBSERVER/total.mdl:31]: sure, will do.

File: genie/MODEL/SPECIFIC/OBSERVER/total.mdl (hunk 2 above)
Comment (gautvenk) [reply genie/MODEL/SPECIFIC/OBSERVER/total.mdl:4]: okay

**Pull request merged**